client/less/main.less

```diff
@@ -300,6 +300,8 @@ ul {
   }
 }
 
+li.avatar, li.avatar > a { padding:0; margin:0 }
+
```

Comment:
For future reference, each style expression should be on it's own line, there should be a space after the colon, and all style expressions should end with a semicolons.
```css
li.avatar, li.avatar > a {
  padding: 0;
  margin: 0;
}
```